sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/implementation/metricsbatch/AzureMonitorMetricBatchBuilder.java

```diff
@@ -268,7 +268,7 @@ private HttpPipeline createHttpPipeline() {
         policies.add(new AddDatePolicy());
         policies.add(new CookiePolicy());
         if (tokenCredential != null) {
-            policies.add(new BearerTokenAuthenticationPolicy(tokenCredential, String.format("%s/.default", endpoint)));
+            policies.add(new BearerTokenAuthenticationPolicy(tokenCredential, "https://metrics.monitor.azure.com/.default"));
```

Comment:
What about other clouds? `.com` just works for the public cloud.

Reply:
Yeah, we'll add support to other clouds when we get the details of the audience from the service team.